task/sast-go/0.1/sast-go.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
   name: sast-go
 spec:
   description: >-
-    Static application security testing for Go code.
+    Uses GoSec, a Static Application Security Testing (SAST) tool for identifying security issues in Go source code.
```

Comment:
Suggestion: Uses GoSec, a Static Application Security Testing (SAST) tool, to identify security issues in Go source code.